protocol/ntpshm/ntpshm.go

```diff
@@ -62,14 +63,22 @@ func Create() (uintptr, error) {
 	return shmID, nil
 }
 
-func ptrToNTPSHM(shmptr uintptr) *NTPSHM {
-	var shm []NTPSHM
-	sh := (*reflect.SliceHeader)(unsafe.Pointer(&shm))
-	sh.Data = shmptr
-	sh.Len = 1
-	sh.Cap = 1
+func ptrToBytes(shmptr uintptr) []byte {
+	// Slice memory layout
```

Comment:
NIT: expand comment to make it more clear that this is a "runtime representation of a slice in Go"